xds/internal/xdsclient/clientimpl_watchers.go

```diff
@@ -34,7 +34,7 @@ type wrappingWatcher struct {
 }
 
 func (w *wrappingWatcher) OnError(err error, done xdsresource.OnDoneFunc) {
-	w.ResourceWatcher.OnError(fmt.Errorf("[xDS node id: %v]: %v", w.nodeID, err), done)
+	w.ResourceWatcher.OnError(fmt.Errorf("[xDS node id: %v]: %w", w.nodeID, err), done)
```

Comment:
Funny. I have the same change here and in `xds/internal/xdsclient/xdsresource/errors.go` as part of my current PR to add xDS node ID from errors generated by the cds LB policy and the xDS resolver. But you'll definitely beat me to the merge line.